packages/api/src/utils/db-types.d.ts

```diff
@@ -857,6 +869,16 @@ export interface definitions {
     /** Format: timestamp with time zone */
     deleted_at?: string
   }
+  upload_stats: {
+    /**
+     * Format: bigint
+     * @description Note:
+     * This is a calculated value on a materialized view that represents a whole number percent
+     */
+    total_uploads: number
+    total_deals: number
+    growth_rate_percent: number
```

Comment:
Where is this used?

Reply:
This is the returned value from the stats page, growth_rate needs removed, I'll get it updated.